docs/make.jl

```diff
@@ -1,17 +1,17 @@
 using Documenter, Syslogs
 
 makedocs(
-    modules=[Syslogs],
-    format=:html,
-    pages=[
+    modules = [Syslogs],
+    format = Documenter.HTML(prettyurls = get(ENV, "CI", nothing) == "true"),
+    pages = [
         "Home" => "index.md",
     ],
-    repo="https://github.com/invenia/Syslogs.jl/blob/{commit}{path}#L{line}",
-    sitename="Syslogs.jl",
-    authors="Invenia Technical Computing Corporation",
-    assets=[
+    repo = "https://github.com/invenia/Syslogs.jl/blob/{commit}{path}#L{line}",
+    sitename = "Syslogs.jl",
+    authors = "Invenia Technical Computing Corporation",
+    assets = [
         "assets/invenia.css",
-     ],
+    ],
```

Comment:
I was going to comment that I thought the standard was no spaces around the `=` sign, but looking at other projects, it seems like we do both in the same file in various projects.

Reply:
Ya, I'm not sure why this file seems to be special